### PR TITLE
Address genart comments

### PR DESCRIPTION
**Pull request by tfpauly**
https://datatracker.ietf.org/doc/review-ietf-intarea-provisioning-domains-09-genart-lc-dupont-2019-12-27/

### CODE REVIEW

**Review by ekline — APPROVED** [review]
LGTM, FWIW  :-)

**Comment by Oryon** [comment]
LGTM as well. Thanks !

File: draft-ietf-intarea-provisioning-domains.md [hunk 1]
```diff
@@ -119,7 +119,10 @@ the PvD ID Router Advertisement option which, when present, associates
 the PvD ID with all the information present in the Router Advertisement
 as well as any configuration object, such as addresses, derived from
 it. The PVD ID Router Advertisement option may also contain a set of
-other RA options. These options are only visible to 'PvD-aware' hosts.
+other RA options, along with an optional inner Router Advertisement
+message header. These options and optional header are only visible
```
Comment (evyncke) [inline draft-ietf-intarea-provisioning-domains.md]: s/optional header/optional inner RA header/

Just to be clear (albeit a little redundant...)

**Comment by evyncke** [comment]
Beside my above comment, it is fine with me.

**Review by evyncke — APPROVED** [review]

**Pull request merged**